p2p/pex/errors.go

```diff
@@ -63,3 +63,12 @@ type ErrAddrBookInvalidAddr struct {
 func (err ErrAddrBookInvalidAddr) Error() string {
 	return fmt.Sprintf("Cannot add invalid address %v: %v", err.Addr, err.AddrErr)
 }
+
+// Err is thrown when the address is banned and therefore cannot be used
```

Comment:
Go comments should start with name of what they're describing, i.e. `ErrAddressBanned is thrown ...`. I thought the linter would catch this, but we may have disabled it.